pages/common/chafa.md

```diff
@@ -0,0 +1,25 @@
+# chafa
+
+> Image printing in the terminal.
+> See also: `catimg`, `pixterm`.
+> More information: <https://hpjansson.org/chafa>.
+
+- Render an image or animated GIF directly in the terminal:
```

Comment:
```suggestion
- Render an image or a GIF directly in the terminal:
```

Reply:
I was thinking it might be an important distinction. `pixterm`, for example, can only render static GIFs. But GIF is synonymous with animation for most people, so maybe that isn't so useful. I might just omit the GIF mention entirely.

```suggestion
- Render an image directly in the terminal:
```

Reply:
Yeah, I'm just removing it.

Reply:
to me, a gif is a collection of pictures put together (just described a video :smile: ) so in my definition, it is an animation.

Reply:
Maybe I'm just being too particular then.